modules/admin_manual/pages/installation/quick_guides/ubuntu_22_04.adoc

```diff
@@ -0,0 +1,329 @@
+= Install ownCloud on Ubuntu 22.04
+:toc: right
+:page-aliases: installation/quick_guides/ubuntu_18_04.adoc
```

Comment:
```suggestion
```
This is not needed as it is a new document and we have no legacies to take care of 😄 
Just press commit and rebase you local branch